tests/etl/test_loaders.py

```
@@ -0,0 +1,59 @@
+import pandas as pd
+from types import SimpleNamespace
+from src.etl.loaders import PostgreSQLLoader, TableMappingLoader
+
+
+class MockDBManager:
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)